main/acle.md

```diff
@@ -11225,42 +11225,43 @@ Zero ZT0
 Lookup table read with 2-bit and 4-bit indexes
 
 ``` c
-  // Variants are also available for _zt[_s8], _zt[_u16], _zt[_s16], _zt[_u32]
-  // and _zt[_s32]
-  svuint8_t svluti2_lane_zt[_u8](uint64_t zt, svuint8_t zn, uint64_t imm_idx)
+  // Variants are also available for _zt_u8, _zt_s16, _zt_u16, _zt_f16,
+  // _zt_bf16, _zt_s32, _zt_u32 and _zt_f32
+  svint8_t svluti2_lane_zt_s8(uint64_t zt, svuint8_t zn, uint64_t imm_idx)
     __arm_streaming __arm_shared_za __arm_preserves_za;
 
 
-  // Variants are also available for _zt[_s8], _zt[_u16], _zt[_s16], _zt[_u32]
-  // and _zt[_s32]
-  svuint8x2_t svluti2_lane_zt[_u8]_x2(uint64_t zt, svuint8_t zn,
-                                      uint64_t imm_idx)
+  // Variants are also available for _zt_u8, _zt_s16, _zt_u16, _zt_f16,
+  // _zt_bf16, _zt_s32, _zt_u32 and _zt_f32
+  svint8x2_t svluti2_lane_zt_s8_x2(uint64_t zt, svuint8_t zn,
+                                   uint64_t imm_idx)
     __arm_streaming __arm_shared_za __arm_preserves_za;
 
 
-  // Variants are also available for _zt[_s8], _zt[_u16], _zt[_s16], _zt[_u32]
-  // and _zt[_s32]
-  svuint8x4_t svluti2_lane_zt[_u8]_x4(uint64_t zt, svuint8_t zn,
-                                      uint64_t imm_idx)
+  // Variants are also available for _zt_u8, _zt_s16, _zt_u16, _zt_f16,
+  // _zt_bf16, _zt_s32, _zt_u32 and _zt_f32
+  svint8x4_t svluti2_lane_zt_s8_x4(uint64_t zt, svuint8_t zn,
+                                   uint64_t imm_idx)
     __arm_streaming __arm_shared_za __arm_preserves_za;
 
 
-  // Variants are also available for _zt[_s8], _zt[_u16], _zt[_s16], _zt[_u32]
-  // and _zt[_s32]
-  svuint8_t svluti4_lane_zt[_u8](uint64_t zt, svuint8_t zn, uint64_t imm_idx)
+  // Variants are also available for _zt_u8, _zt_s16, _zt_u16, _zt_f16,
+  // _zt_bf16, _zt_s32, _zt_u32 and _zt_f32
+  svint8_t svluti4_lane_zt_s8(uint64_t zt, svuint8_t zn, uint64_t imm_idx)
     __arm_streaming __arm_shared_za __arm_preserves_za;
 
 
-  // Variants are also available for _zt[_s8], _zt[_u16], _zt[_s16], _zt[_u32]
-  // and _zt[_s32]
-  svuint8x2_t svluti4_lane_zt[_u8]_x2(uint64_t zt, svuint8_t zn,
-                                      uint64_t imm_idx)
+  // Variants are also available for _zt_u8, _zt_s16, _zt_u16, _zt_f16,
+  // _zt_bf16, _zt_s32, _zt_u32 and _zt_f32
+  svint8x2_t svluti4_lane_zt_s8_x2(uint64_t zt, svuint8_t zn,
+                                   uint64_t imm_idx)
     __arm_streaming __arm_shared_za __arm_preserves_za;
 
 
-  // Variants are also available for _zt[_s16], _zt[_u32] and _zt[_s32]
-  svuint16x4_t svluti4_lane_zt[_u16]_x4(uint64_t zt, svuint16_t zn,
-                                        uint64_t imm_idx)
+  // Variants are also available for _zt_u16, _zt_f16, _zt_bf16, _zt_s32,
+  // _zt_u32 and _zt_f32
+  svint16x4_t svluti4_lane_zt_s16_x4(uint64_t zt, svuint16_t zn,
```

Comment:
Hi @rsandifo-arm,
Should the zn argument for the svluti4 x4 also be a svuint8_t?